Packs/CortexVulnerabilityManagement/Playbooks/Cortex_VM_-_Vulnerability_Issue.yml

```diff
@@ -691,20 +684,22 @@ tasks:
         simple: ${Core.CoreAsset.xdm__asset__provider}
       val4:
         simple: TIMESTAMP
+      context_key:
+        simple: vmassetownerunrankedraw
     separatecontext: false
     skipunavailable: false
     task:
       brand: Builtin
       description: |-
-        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. For example:
-        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
-      id: 86485c5d-e273-41cb-a86b-b46a8640cd35
+        Automation used to more easily populate XSOAR context.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format. For example:
```

Comment:
```suggestion
        Automation used to more easily populate Cortex XSOAR context. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format. For example:
```